### ISSUE TITLE
webapi prints DB user and password on console on startup

### ISSUE THREAD

**Issue by benjamingeer** [dhlab-basel/Knora#789]
Since merging from `develop`, when I start `webapi`, it prints:

```
webapi DB Server: localhost, DB Port: 7200
webapi Repository: knora-test
webapi DB User: admin
webapi DB Password: root
webapi Knora API Server started at http://0.0.0.0:3333.
```

Is this intentional?

**Comment by subotic** [comment]
yes, intentional :-) lost hours wondering why I could not connect to a secured GraphDB. turned out to be an error in the config, so the wrong password was used. one day I will make it configurable, so that only if a flag is set in `application.conf` key properties are printed out.

**Comment by benjamingeer** [comment]
I‘m concerned that this is a security risk. On a production system, the log output is likely to be saved to a file. If the sysadmin doesn‘t think to make the log file permissions restrictive enough, the database password could be readable by any process on the system.

**Comment by benjamingeer** [comment]
Also, I think passwords shouldn‘t be printed to the console. Someone could be looking over the sysadmin‘s shoulder, or someone could copy and paste the log output into an email and not notice that the password is in it. I think good practice is never to display a password unless an authorised user explicitly asks for it. So I think it would be OK to make this a configuration option, but not to make it the default.

**Comment by subotic** [comment]
> So I think it would be OK to make this a configuration option, but not to make it the default.

Changed the title of the issue. Will be implemented soon (TM) :-)
